extensions/amp-story/1.0/amp-story.css

```diff
@@ -450,6 +450,13 @@ amp-story-grid-layer .i-amphtml-embedded-component::after {
                        "lower-third" !important;
 }
 
+.i-amphtml-story-grid-template-aspect {
+  margin: auto;
+  width: var(--i-amphtml-story-layer-width, 100%);
+  height: var(--i-amphtml-story-layer-height, 100%);
+  font-size: calc(var(--i-amphtml-story-layer-height, 100vh) / 10);
```

Comment:
So, am I reading this correctly that the default font-size is 1/10 the height of the root container?

Reply:
Yes. That's the suggestion. See the description of the PR for how I got here.